subsys/testsuite/ztest/Kconfig

```diff
@@ -181,6 +181,58 @@ config TEST_LOGGING_FLUSH_AFTER_TEST
 	default y
 	depends on MULTITHREADING
 
+config ZTEST_MAIN
+	bool "Run ZTest test suites from main"
+	default y
+	help
+	  When disabled, ZTest test suites are not automatically run from main().
+
+config ZTEST_SHELL
+	bool "ZTest shell command"
+	select SHELL
+	select GETOPT
+	select FNMATCH
+	select GETOPT_LONG
+	select SHELL_GETOPT
+	# currently this is broken due to getopt_state / conflicting native definitions on POSIX arch
+	depends on !ARCH_POSIX
+	help
+	  Provide a shell command to run ZTests from the Zephyr shell. The arguments of the 'ztest'
```

Comment:
ztest and the zephyr shell can do more than what gtest does or provides, so this is rather limiting, i.e. I would like to see feature of the shell being used, such as sub commands, auto-completion etc  instead of depending on command lines for everything.
Compatibility mode with gtest is fine, but not as the main shell implementation.

Reply:
Subcommands are fine for a specific class of command. Here, flags make a bit more sense. There is no reason to use subcommands simply because they exist.